src/configuration-parser.ts

```diff
@@ -19,6 +19,8 @@ export class ConfigurationParser {
             pending: "* ",
             successful: ConfigurationParser.isWindows ? "\u221A " : "✓ ",
         },
+        // tslint:disable-next-line:no-unbound-method
+        print: console.log,
```

Comment:
TestHelper needs to mock the default print feature so the (non-custom-print) tests can examine the reporter output. ConfigurationParser needs a default print feature. So we need some place where TestHelper and ConfigurationParser can agree to use for the default print feature.  They current agree on the global "console.log".

We could use another global, or we could add a static to any class.  TestHelper needs to access that wrapper to overwrite it for mocking.

I took a stab, is this what you have in mind?

The other way to go is to have TestHelper overwrite this.reporter.print